src/app/pipes/CapitalizePipe.ts

```diff
@@ -13,15 +13,18 @@ export class CapitalizePipe implements Pipe {
   supports(txt): boolean {
     return isString(txt);
   }
-  transform(input: string, isEveryWord?: boolean): string {
-    return (!input) ? '' :
-      (!isEveryWord) ?
-        this.capitalizeWord(input) :
-        input.replace(this.regexp, this.capitalizeWord);
+  transform(value: string, args: List<any>): any {
```

Comment:
the second argument is either optional or null

`args?: List<any>` or `args: List<any> = null`